src/components/Rating/Rating.driver.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  BaseUniDriver,
+  baseUniDriverFactory,
+} from 'wix-ui-test-utils/base-driver';
+import { StylableUnidriverUtil, UniDriver } from 'wix-ui-test-utils/unidriver';
+import style from './Rating.st.css';
+
+export interface RatingDriver extends BaseUniDriver {
+  isMobile(): Promise<boolean>;
```

Comment:
If there's no `mobile` state for the component, you can remove all the references to the `mobile` state